### DISCUSSION TITLE
404 erroe

### DISCUSSION THREAD

**Discussion by MarioShawky2021** [community/community#33173] [Other Feature Feedback, Questions, & Ideas]
i have 404 error on my page
https://marioshawky2021.github.io/new/
i don't know how to solved
i have HTML, CSS and JS files

**Comment by jfullstackdev** [comment]
since it's not the default name, `index.html`, you are seeing the README 

but if you have this URL 

https://marioshawky2021.github.io/new/SpecialDesign.html

there is your website

![image](https://user-images.githubusercontent.com/47092464/190436765-05e5f82b-9378-4a82-b968-93bf9c66d35e.png)

**Reply by MarioShawky2021** [reply]
thanks :) :)
